tests/instances.spec.ts

```diff
@@ -213,17 +214,13 @@ test("instance yaml edit", async ({ page }) => {
   test.skip(Boolean(process.env.CI), "github runners lack vm support");
   await editInstance(page, vmInstance);
   await page.getByText("YAML configuration").click();
-
   await page.locator(".view-lines").click();
-  await page.keyboard.press("PageDown");
-  await page.keyboard.press("PageDown");
-  await page.keyboard.press("PageDown");
```

Comment:
why was this removed?

Reply:
The test wasn't running with them in. The item that needed to be viewed was not visible after 3 pagedown clicks.